### PR TITLE
[IMP] mail: restore and clean up fields tracking values

### PR DESCRIPTION
**Pull request by seb-odoo**
task-2185139

### CODE REVIEW

File: addons/mail/static/src/owl/components/message/message.js [hunk 1]
```diff
@@ -225,17 +225,9 @@ class Message extends Component {
...
-            return [];
-        }
         return this.storeProps.message.tracking_value_ids.map(trackingValue => {
-            let value = {
-                changed_field: trackingValue.changed_field,
-                old_value: trackingValue.old_value,
-                new_value: trackingValue.new_value,
-                field_type: trackingValue.field_type,
-            };
+            let value = Object.assign({}, trackingValue);
```
Comment (alexkuhn) [inline addons/mail/static/src/owl/components/message/message.js]: `const`

File: addons/mail/static/src/owl/components/message/message.scss [hunk 2]
```diff
@@ -147,6 +147,19 @@
...
+}
+
+.o_Message_trackingValues_value {
+    display: flex;
+    align-items: center;
+}
+
+.o_Message_trackingValues_value_item {
+    margin-inline-end: map-get($spacers, 1);
+}
```
Comment (alexkuhn) [inline addons/mail/static/src/owl/components/message/message.scss]: naming convention: `.o_Message_trackingValue` and `.o_Message_trackingValueItem`

**Review by alexkuhn — CHANGES_REQUESTED** [review]

File: addons/mail/static/src/owl/components/message/message.js (hunk 1 above)
Comment (seb-odoo) [reply addons/mail/static/src/owl/components/message/message.js]: done

File: addons/mail/static/src/owl/components/message/message.scss (hunk 2 above)
Comment (seb-odoo) [reply addons/mail/static/src/owl/components/message/message.scss]: and done

**Review by alexkuhn — APPROVED** [review]

**Pull request merged**